ci-operator/step-registry/ipi/azure/pre/stableinitial/ipi-azure-pre-stableinitial-chain.metadata.json

```diff
@@ -2,7 +2,12 @@
 	"path": "ipi/azure/pre/stableinitial/ipi-azure-pre-stableinitial-chain.yaml",
 	"owners": {
 		"approvers": [
-			"abhinavdahiya"
+			"vrutkovs",
+			"deads2k",
+			"technical-release-team-approvers",
```

Comment:
are they aware?

Reply:
After some back-channel discussion, all of the other parent-directory approvers are ok with being direct approvers in the three previously-approver-less directories, except for @deads2k who who did not want to become a direct approver in any of the three previously-approver-less directories.  I've rerolled to reflect that with 95f97b7e534 -> cb6698cf7dc.